asmcomp/selectgen.ml

```diff
@@ -42,6 +42,10 @@ let oper_result_type = function
   | Cintoffloat -> typ_int
   | Craise _ -> typ_void
   | Ccheckbound _ -> typ_void
+  | Cperform -> typ_addr
```

Comment:
Is the result type of `Cperform`, `Cresume` and `Cdelegate` always typ_addr? Should they be parameterized with the result type similar to `Capply`?